osu.Game/Graphics/Containers/OsuRearrangeableListContainer.cs

```diff
@@ -18,11 +23,44 @@ public abstract class OsuRearrangeableListContainer<TModel> : RearrangeableListC
 
         protected override ScrollContainer<Drawable> CreateScrollContainer() => new OsuScrollContainer();
 
+        private Sample sampleSwap;
+        private double sampleLastPlaybackTime;
+
         protected sealed override RearrangeableListItem<TModel> CreateDrawable(TModel item) => CreateOsuDrawable(item).With(d =>
         {
             d.DragActive.BindTo(DragActive);
         });
 
         protected abstract OsuRearrangeableListItem<TModel> CreateOsuDrawable(TModel item);
+
+        protected OsuRearrangeableListContainer()
+        {
+            Items.CollectionChanged += (_, args) =>
+            {
+                if (args.Action == NotifyCollectionChangedAction.Move)
+                    playSwapSample();
+            };
+        }
+
+        private void playSwapSample()
+        {
+            if (Time.Current - sampleLastPlaybackTime <= 35)
```

Comment:
What's the case where you could make it play faster than this out of curiosity?

Reply:
Dragging items around in a list while also scrolling the list is one particularly bad case...

with the debounce:

https://user-images.githubusercontent.com/272140/175283777-c71e5509-73c5-4af7-9ed4-ca1ca69400ea.mov

vs without:

https://user-images.githubusercontent.com/272140/175283838-2ebacd44-292e-400c-9534-18108d71f527.mov

Could probably also just do it by dragging fast enough, but the above case is pretty egregious.